online-dashboard-api/database/migrations/2024_12_24_073310_create_job_opportunities_table.php

```diff
@@ -13,11 +13,11 @@ public function up(): void
     {
         Schema::create('job_opportunities', function (Blueprint $table) {
```

Comment:
dont modify the old migration
if you need to do any update on the previous migration  - we need write a new migration

Reply:
and this fields are modified in later updates of migrations- do check and modify accoridingly